backend/api/src/create-market.ts

```diff
@@ -625,4 +646,30 @@ async function generateAntes(
 
     await insertLiquidity(pg, lp)
   }
+      const drizzledAmount = totalMarketCost - ante 
+      if (drizzledAmount > 0) {
+          return await pg.tx(async (tx) => {
+            await runTxn(tx, {
+              fromId: providerId,
+              amount: drizzledAmount,
+              toId: contract.id,
+              toType: 'CONTRACT',
+              category: 'ADD_SUBSIDY',
+              token: 'M$',
+              fromType: 'USER',
+            })
+                const newLiquidityProvision = getNewLiquidityProvision(
+      providerId,
+      drizzledAmount,
+      contract
+    )
+
+    await insertLiquidity(tx, newLiquidityProvision)
+
+    contractRef.update({
+      subsidyPool: FieldValue.increment(drizzledAmount),
+      totalLiquidity: FieldValue.increment(drizzledAmount),
+    })
+  })
+}
```

Comment:
I think this is right for binary, but for multiple choice, we want to split the drizzled amount evenly among answers, insert a liquidity doc for each, and then update the answer doc (`contracts/[contractId]/answersCpmm/[answerId]`) with subsidyPool and totalLiquidity.

Reply:
Is there a precedent for this? I just reused whatever was in the add subsidy code, does that not do that?

Reply:
Oh, I guess either works for now, so this is fine.

If this is exactly the same functionality as in addContractLiquidity, you might want to extract a helper function that is used in both places. Would make it much easier to update both in the future.

Reply:
Yeah I thought about that but most of the stuff is already extracted into a function and it's dissimilar enough that I think it just wouldn't make sense